sourcemash/frontend/templates/register.tpl

```diff
@@ -0,0 +1,21 @@
+<h2>Sign Up</h2>
```

Comment:
Good catch! Will delete file

Scott

From: Alex Gerstein <notifications@github.com>
Reply: sourcemash/Sourcemash <reply@reply.github.com>>
Date: April 28, 2015 at 5:17:40 PM
To: sourcemash/Sourcemash <sourcemash@noreply.github.com>>
Cc: Scott Gladstone <sgladstone11@gmail.com>>
Subject:  Re: [Sourcemash] Modal sign up (#221)  

In sourcemash/frontend/templates/register.tpl:

> @@ -0,0 +1,21 @@
> +<h2>Sign Up</h2>
Do we still use the register page? Or is everything on the modal now?

—
Reply to this email directly or view it on GitHub.